apps/shared/test-helpers/mine.js

```diff
@@ -0,0 +1,43 @@
+const Web3 = require("web3");
+const PrivateKeyProvider = require("truffle-privatekey-provider");
+const provider = new PrivateKeyProvider(
```

Comment:
If we only execute this script through `truffle exec` in npm or in a testing context we could probably just infer the provider. I'm fine with leaving this as-is though.